### ISSUE TITLE
Accent color

### ISSUE THREAD

**Issue by yephny** [tberghuis/FloatingCountdownTimer#4]
Allow custom accent color to be used instead of preset dark blue.

**Comment by Ilithy** [comment]
The best would be to use Monet (Material You) to have a dynamically accentuated coloring, it is classy...

**Comment by yephny** [comment]
Wouldn't that be incompatible with older Android versions? @Ilithy

**Comment by Ilithy** [comment]
@yephny Indeed, MY is only supported from android 12 and newer if I'm not mistaken...I understand my mistake 😅 so the two are complementary.
